api-compact.yaml

```diff
@@ -151,6 +195,37 @@ x-amazon-apigateway-integration-with-id-and-querystring-parameters: &apigateway_
     integration.request.querystring.page: "method.request.querystring.page"
     integration.request.querystring.page_size: "method.request.querystring.page_size"
 
+x-amazon-apigateway-integration-uuid-response: &apigateway_integration_uuid
```

Comment:
@saleem-latif See above comment -- `uuid` is also used by the new `enterprise_catalogs/{uuid}` endpoint, so this may be confusing.